generators/server/templates/src/main/java/package/config/_Constants.java

```diff
@@ -8,6 +8,7 @@
     //Regex for acceptable logins
     public static final String LOGIN_REGEX = "^[_'.@A-Za-z0-9-]*$";
     // Spring profile for development and production, see http://jhipster.github.io/profiles/
+    public static final String SPRING_PROFILE_TEST = "test";
```

Comment:
Yes, that was my initial idea, but then the problem is you can not access the `DatabaseDescriptor.getNativeTransportPort();` method.
It's part of the cassandra-all lib, and only visible in the test scope.
Which makes sense because it's pulled by cassandra-unit only for the tests.

I did not find another way to discover the port from the java driver.

The only alternative I can think of is to have AbstractCassandraTest modifying the configuration properties passed before the Spring context is started. But I have no clue how to do this.